esx_service/cli/vmdkops_admin.py

```diff
@@ -236,9 +236,8 @@ def commands():
                     'func': tenant_rm,
                     'help': 'Delete a tenant',
                     'args': {
-                        '--name': {
+                        'name': {
```

Comment:
why is it changed for 'tenant rm' ? I see 'tenant create' and 'tenant update' are stil using `--name` and 'tenant rm' now uses positional 'name'. This is inconsistent. 

Please drop this change, and instead use #922 to enumerate all changes to --name or name whihc are needed to make all consistent. See #922 for more comments